src/dvr/dvr_db.c

```diff
@@ -17,6 +17,8 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define _ISOC9X_SOURCE
```

Comment:
Here the error:
> CC              src/dvr/dvr_db.o
> cc1: warnings being treated as errors
> src/dvr/dvr_db.c: In function 'dvr_entry_fuzzy_match':
> src/dvr/dvr_db.c:839: error: implicit declaration of function 'llabs'
> src/dvr/dvr_db.c:839: error: incompatible implicit declaration of built-in function 'llabs'
> Makefile:627: recipe for target '/home/me/builds/spksrc/spk/tvheadend/work-qoriq-5.2/tvheadend-master/build.linux/src/dvr/dvr_db.o' failed
> make[3]: *** [/home/me/builds/spksrc/spk/tvheadend/work-qoriq-5.2/tvheadend-master/build.linux/src/dvr/dvr_db.o] Error 1
> make[3]: Leaving directory '/home/me/builds/spksrc/spk/tvheadend/work-qoriq-5.2/tvheadend-master'
> ../../mk/spksrc.compile.mk:36: recipe for target 'compile_target' failed
> make[2]: *** [compile_target] Error 2
> make[2]: Leaving directory '/home/me/builds/spksrc/cross/tvheadend'
> ../../mk/spksrc.depend.mk:44: recipe for target 'depend_target' failed
> make[1]: *** [depend_target] Error 2
> make[1]: Leaving directory '/home/me/builds/spksrc/spk/tvheadend'
> ../../mk/spksrc.spk.mk:418: recipe for target 'arch-qoriq-5.2' failed
> make: [arch-qoriq-5.2] Error 2 (ignored)

This error actually was an issue with TVH before. See [here](https://tvheadend.org/issues/1634) for how it was fixed then. Same fix here...